File: Clojure/start-code/startcode.clj
(ns startcode.core)

(defn -main []
  (println "Willkommen zu meinem Clojure-Programm!")

  (print "Bitte geben Sie Ihren Namen ein: ")
  (flush) ; Sorgt dafür, dass der Prompt sofort angezeigt wird
  (let [name (read-line)] ; Liest die Eingabe des Nutzers
    (println (str "Hallo, " name "! Schön, dass Sie hier sind.")))

  (println "Das Programm wurde beendet."))

(-main)

File: F-Sharp/start-code/startcode.fs
open System

// Begrüßung
Console.WriteLine("Willkommen zu meinem F#-Programm!")

// Nutzereingabe
Console.Write("Bitte geben Sie Ihren Namen ein: ")
let name = Console.ReadLine()

// Begrüßung mit dem Namen
Console.WriteLine($"Hallo, {name}! Schön, dass Sie hier sind.")
Console.WriteLine("Das Programm wurde beendet.")
